packages/xod-cli/src/xodc-install.js

```diff
@@ -3,17 +3,20 @@ import * as xodFs from 'xod-fs';
 import { fetchLibData, fetchLibsWithDependencies } from 'xod-pm';
 import * as messages from './messageUtils';
 import * as MSG from './messages';
+import { getWorkspacePath } from './utils';
 
 /**
  * Installs the library version from package manager.
  * @param {LibUri} libQuery - Library identifier in the package manager (`owner/name@version`).
- * @param {Path} distPath - Path to workspace, where should be library installed
+ * @param {Path} workspace - Path to workspace, where should be library installed
  * @return {Promise.<void>}
  */
-export default function install(swaggerUrl, libQuery, distPath) {
+export default function install(swaggerUrl, libQuery, workspace) {
   return Promise.all([
     fetchLibData(swaggerUrl, libQuery),
-    xodFs.findClosestWorkspaceDir(distPath),
+    xodFs.findClosestWorkspaceDir(
+      getWorkspacePath(workspace)
+    ),
```

Comment:
So we pass a `Path` here, which is defaulted to a default workspace path if it's `null`, and then scan that path and it's parent directories for `.xodworkspace`🤔
Do we really need it? I think something like `R.compose(ensureThatItsAWorkspace, getWorkspacePath)(workspace)` should be enough.